Week02/lesson04/homework/layoutChallenge/3/index.html

```diff
@@ -0,0 +1,54 @@
+ <!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Layout three</title>
+
+  <!-- external CSS links -->
+	<link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+    <div id="container">
+
+      <header class="block">
+
+      </header>
+
+
+      <div class="clearfix">
+
+          <article class="block left">
+
+          </article>
+
+          <article class="block right">
+
+          </article>
+
+
+          <div class="boxOne block">
```

Comment:
I would consider using two `clearfix`'ed parents for each row in this layout. Maybe `<main class="clearfix">` would wrap around the two `article`'s and then `<section class="clearfix">` could  wrap around the three `.box`'es, but that's just a matter of preference!